srcts/src/shiny/shinyapp.ts

```diff
@@ -1395,17 +1431,24 @@ class ShinyApp {
         // Stack bars
         const $progressBar = $progress.find(".progress");
 
-        $progressBar.css("top", depth * $progressBar.height() + "px");
+        if ($progressBar) {
+          $progressBar.css(
+            "top",
+            depth * ($progressBar.height() as number) + "px"
+          );
 
-        // Stack text objects
-        const $progressText = $progress.find(".progress-text");
+          // Stack text objects
+          const $progressText = $progress.find(".progress-text");
 
-        $progressText.css(
-          "top",
-          3 * $progressBar.height() + depth * $progressText.outerHeight() + "px"
-        );
+          $progressText.css(
+            "top",
+            3 * ($progressBar.height() as number) +
+              depth * ($progressText.outerHeight() as number) +
+              "px"
+          );
 
-        $progress.hide();
+          $progress.hide();
```

Comment:
This logic is moved inside the `if($progressBar)`. Didn't make sense to hide something that didn't exist